src/components/Posts/ComposeReply.tsx

```diff
@@ -71,6 +75,9 @@ const ComposeReply = ({ placeholder }: ComposeReplyProps) => {
           displayName: user.displayName,
         })
       );
+      if (onClose) {
+        onClose();
+      }
```

Comment:
I believe you can simplify this using optional chaining. Can read more about it [here](https://javascript.info/optional-chaining)

```suggestion
      onClose?.()
```

Reply:
Nice suggestion! I didn't know about this and was annoying me